flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlDdlToOperationConverterTest.java

```diff
@@ -658,6 +659,37 @@ public void testCreateTableInvalidDistribution() {
                         "Invalid bucket key 'f3'. A bucket key for a distribution must reference a physical column in the schema. Available columns are: [a]");
     }
 
+    @Test
+    public void testExplainCreateTableAs() {
```

Comment:
I don't think we need this test. The explain test below that outputs the full pipeline should be sufficient.

Reply:
Done